x/intrarelayer/keeper/msg_server_test.go

```diff
@@ -271,3 +272,28 @@ func (suite *KeeperTestSuite) TestConvertCoinNativeERC20() {
 	}
 	suite.mintFeeCollector = false
 }
+
+func (suite *KeeperTestSuite) TestConvertNativeIBC() {
+	suite.SetupTest()
+	validMetadata := banktypes.Metadata{
+		Description: "desc",
+		Base:        "ibc/7F1D3FCF4AE79E1554D670D1AD949A9BA4E4A3C76C63093E17E446A46061A7A2 channel-14",
+		// NOTE: Denom units MUST be increasing
+		DenomUnits: []*banktypes.DenomUnit{
+			{
+				Denom:    "ibc/7F1D3FCF4AE79E1554D670D1AD949A9BA4E4A3C76C63093E17E446A46061A7A2 channel-14",
+				Exponent: 0,
+			},
+			{
+				Denom:    "coin2",
+				Exponent: uint32(18),
+			},
+		},
+		Name:    "ibc/7F1D3FCF4AE79E1554D670D1AD949A9BA4E4A3C76C63093E17E446A46061A7A2 channel-14",
+		Symbol:  "ibc/7F1D3FCF4AE79E1554D670D1AD949A9BA4E4A3C76C63093E17E446A46061A7A2 channel-14",
+		Display: "ibc/7F1D3FCF4AE79E1554D670D1AD949A9BA4E4A3C76C63093E17E446A46061A7A2 channel-14",
```

Comment:
we can't change the display because we don't
```suggestion
		Display: "ibcATOM",
```

Reply:
Display should be == Denom[0] == Base